src/vivarium_public_health/disease/state.py

```diff
@@ -201,6 +199,7 @@ def setup(self, builder):
         disability_weight_data = get_disability_weight_func(self.cause, builder)
         self.prevalence_data = builder.lookup.build_table(get_prevalence_func(self.cause, builder))
         self._dwell_time = get_dwell_time_func(self.cause, builder)
+        self.randomness = builder.randomness.get_stream(f'determine_event_time_of_{self.state_id}_prevalent_cases')
```

Comment:
Hah, the name I meant to make more descriptive is `self.randomness`.  The old stream name was fine.  The attribute name is too generic.